### ISSUE TITLE
Error when updating to 1.0.19

### ISSUE THREAD

**Issue by dboscanv** [angular-ui/ui-router#3726]
- [x] Bug Report
- [ ] Feature Request
- [ ] General Query 

My version of UI-Router is: 1.0.19

# Bug Report

#### Current Behavior:

When updated to 1.0.19 the following error occurs:

`ERROR in ./~/@uirouter/core/lib-esm/index.jsERROR		
2742.	Module build failed: error: couldn't process source due to parse error,'import' and 'export' may appear only with 'sourceType: module' (5:0)		
2743.	 @ ./~/@uirouter/angularjs/release/ui-router-angularjs.js 9:104-129		
2744.	 @ ./app/index.js`

And the build fails.

#### Expected Behavior:

The code should works. After downgrading to 1.0.18 my code works.

**Comment by schmkr** [comment]
I also noted another issue, and that is that this patch release now uses `Array.prototype.find` which is not (natively) supported by IE11.

**Comment by mattdsteele** [comment]
This has been fixed in core, but is waiting on https://github.com/ui-router/core/commit/66a3244271427942b403931d95118f4d5cc9b509 to ship.

See https://github.com/ui-router/core/issues/215
